### PR TITLE
Fix k8s config

### PR DESCRIPTION
**Pull request by flands**
Addresses #1019

Also add debug logging to local config YAML

### CODE REVIEW

File: examples/k8s.yaml [hunk 1]
```diff
@@ -115,7 +115,7 @@ data:
       batch:
       memory_limiter:
         # Same as --mem-ballast-size-mib CLI argument
-        ballast_size_mib: 165
+        ballast_size_mib: 683
         # 80% of maximum memory up to 2G
         limit_mib: 400
```
Comment (dmitryax) [inline examples/k8s.yaml:120]: `limit_mib` seems to be very low for 2G memory limit, same for `spike_limit_mib`. looks like copied over from the agent config
Should we adjust them as well?

**Comment by owais** [comment]
While this is a good hot fix, I think we should fix this is code so memory limiter automatically picks up whatever size is configured for memory ballast.

**Comment by owais** [comment]
Also, we should see if we move ballast to config instead of being a CLI arg.

**Comment by flands** [comment]
or this: https://github.com/open-telemetry/opentelemetry-collector/issues/1078. These are separate issues though -- goal of this PR is to fix what we have today and is incorrect.

File: examples/k8s.yaml (hunk 1 above)
Comment (flands) [reply examples/k8s.yaml:120]: ah, will fix.

**Pull request merged**